### PR TITLE
3.2 upgrade

### PR DESCRIPTION
**Pull request by BitOne**
**Description (for Contributor and Core Developer)**

Updates the UPGRADE.md file for 3.2

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | Done
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -28,129 +26,92 @@ Please provide a server with the following requirements before proceeding to the
...
+    The only change is the addition of a new container `object-storage`, based on `minio` (see https://min.io/),
+    and compatible with the Amazon S3 protocol. This container allows testing object storage configuration for assets and media.
 
-    TODO: ADD ALL THE CHANGES HERE
 
-6. Update your **app/config/config_behat.yml** and **app/config/config_test.yml**:
+4. Update your **app/config/config.yml**
 
-    An easy way to update it is to copy/paste from the latest standard edition and add your custom changes.
+    The **app/config/config.yml** file didn't change between 3.1 and 3.2.
```
Comment (jjanvier) [inline UPGRADE.md]: can't we remove those lines if the files didn't change?

File: UPGRADE.md [hunk 2]
```diff
@@ -163,61 +124,89 @@ Before updating the dependencies and migrating your data, please deactivate all
     If you have custom code in your project, this step may raise errors in the "post-script" command.
     In this case, go to the chapter "Migrate your custom code" before running the database migration.
 
-    And we also have to update the frontend dependencies:
+10. Update your JS dependencies:
 
-    ```bash
-    yarn install
-    ```
+    The JS dependencies did not change between 3.1 and 3.2.
```
Comment (jjanvier) [inline UPGRADE.md]: updates by Romain

File: CHANGELOG-3.2.md [hunk 3]
```diff
@@ -6,218 +6,44 @@
...
 - Change constructor of `Akeneo\Platform\Bundle\ImportExportBundle\Controller\JobExecutionController` to add `League\Flysystem\FilesystemInterface`
 - Make method `getRealPath` of `Akeneo\Tool\Bundle\BatchBundle\Monolog\Handler\BatchLogHandler` private
 - Change constructor of `Akeneo\Pim\Structure\Component\Validator\Constraints\AttributeTypeForOptionValidator` to add array `$upportedAttributeTypes`
 - Change constructor of `Akeneo\Pim\Enrichment\Component\Product\Job\DeleteProductsAndProductModelsTasklet` to add `Akeneo\Pim\Enrichment\Component\Product\ProductModel\Query\CountProductModelsAndChildrenProductModelsInterface` and `Akeneo\Pim\Enrichment\Component\Product\ProductAndProductModel\Query\CountVariantProductsInterface`
 
+- Rename `Akeneo\Pim\Enrichment\Bundle\Elasticsearch\Filter\Field\AncestorFilter` to `Akeneo\Pim\Enrichment\Bundle\Elasticsearch\Filter\Field\AncestorIdFilter`
+- Rename `Akeneo\Pim\Enrichment\Bundle\Storage\ORM\Connector\GetConnectorProductModels` to `Akeneo\Pim\Enrichment\Bundle\Storage\Sql\Connector\SqlGetConnectorProductModels`
```
Comment (ahocquard) [inline CHANGELOG-3.2.md]: it's a move + rename

File: CHANGELOG-3.2.md [hunk 4]
```diff
@@ -6,218 +6,44 @@
...
+- The following classes and their service definitions have been removed:
+  - `Akeneo\Pim\Enrichment\Bundle\EventSubscriber\RemoveUserSubscriber`
+  - `Akeneo\Pim\Enrichment\Bundle\Storage\ORM\Connector\GetConnectorProductModels`
+  - `Akeneo\Pim\Enrichment\Bundle\Storage\ORM\Connector\GetConnectorProductsFromWriteModel`
+  - `Akeneo\Pim\Enrichment\Bundle\Storage\ORM\Connector\GetMetadataForProductModel`
+  - `Akeneo\Pim\Enrichment\Component\Product\Factory\ValueCollectionFactory`
+  - `Akeneo\Pim\Enrichment\Component\Product\Factory\ValueCollectionFactoryInterface`
+  - `Akeneo\Pim\Enrichment\Component\Product\Model\ValueCollection`
```
Comment (ahocquard) [inline CHANGELOG-3.2.md]: it's WriteValueCollection instead I think

File: CHANGELOG-3.2.md (hunk 3 above)
Comment (BitOne) [reply CHANGELOG-3.2.md]: A move is a rename ;)

File: CHANGELOG-3.2.md (hunk 4 above)
Comment (BitOne) [reply CHANGELOG-3.2.md]: Yes, I've put the new name in the UPGRADE.md file.

File: UPGRADE.md (hunk 1 above)
Comment (BitOne) [reply UPGRADE.md]: I prefer to keep the same structure so next time we can start from this file.

**Pull request merged**